config/configuration/bundles/src/bundle/org/sakaiproject/config/bundle/default.sakai.properties

```diff
@@ -3582,10 +3582,6 @@
 # DEFAULT: "" (no link shown)
 # samigo.recommendations.path=/content/group/RecForStudentsTest.html
 
-# A list of JAR files separated by ":". Each JAR file plugin can provide one or more secure module implementation(s).
-# DEFAULT: none (null)
-# samigo.secureDeliveryPlugins=
```

Comment:
Can i know why you removed this property? Maybe unused?

Reply:
Ahh I see, moved to a different section, nice.

Reply:
Yeah, just a cleanup